src/theme/colors/colors.ts

```diff
@@ -48,11 +48,28 @@ export const common = {
   white: WHITE
 };
 
-export const keppel = {};
+export const keppel = {
+  80: '#daf3eb',
+  70: '#00B39F',
+  60: '#93E6D1',
+  50: '#41CCB3',
+  40: '#00B39F',
+  30: '#008c81',
+  20: '#006661',
+  10: '#00403f'
+};
 
 export const carribean = {};
```

Comment:
@Rexford74  do we not have a tonal palette for Caribbean

Reply:
i guess we have, i will see and update if its there